MetaMorpheus/EngineLayer/PeptideSpectralMatch.cs

```diff
@@ -189,6 +195,14 @@ public void SetFdrValues(double cumulativeTarget, double cumulativeDecoy, double
             };
         }
 
+        public void SetQandPEPvalues(double qValue, double qValueNotch, double pep, double pepQValue)
```

Comment:
Will the other fields in FdrInfo need to be updated as well?

Reply:
see comment in revised PR